packages/design-system/.storybook/main.js

```diff
@@ -1,80 +1,73 @@
 const path = require("path");
 const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin');
-
-async function webpackConfig(config) {
-
-  config.module.rules.find(
-    (rule) => rule.test.toString() === '/\\.css$/'
-  ).exclude = /\.module\.css$/
-
+async function webpackConfig(config, options) {
+  config.module.rules.find(rule => rule.test.toString() === '/\\.css$/').exclude = /\.module\.css$/;
   config.module.rules.push({
```

Comment:
you can use the `@storybook/addon-styling` instead of doing this manually.

Reply:
I have deleted this rule, thanks.

Reply:
No No. I didn't ask to delete the rule. But was recommending that you can use the style addon instead of that. 
Anyway, if you are removing, just make sure it works as expected.

Reply:
We are using addon-styling! Hence deleted the rule :)